pom.xml

```diff
@@ -127,7 +127,7 @@
       <version.maven.jar.plugin>2.4</version.maven.jar.plugin>
       <version.micrometer>1.1.4</version.micrometer>
       <hamcrest.version>2.1</hamcrest.version>
-      <surefire.version>2.22.2</surefire.version>
+      <surefire.version>3.0.0-M5</surefire.version>
```

Comment:
Why are you changing the surefire version?

Reply:
@brusdev 
Yes, I agree with you and verified that in this package, surefire update is not required.